server/schema/squiggle/squiggle.datasources.js

```diff
@@ -41,11 +41,26 @@ const create = async (squiggleType, content) => {
   }
 };
 
+const updateContent = async (id, newContent) => {
+  try {
+    const _squiggle = await SquiggleModel.findById(id);
```

Comment:
@Srish-ty just add a error handling here  if we get null, in case the id provided by the user is invalid.